### PR TITLE
Add backend info to TypeInstance details

### PR DESCRIPTION
**Pull request by mszostok**
## Description

Changes proposed in this pull request:

-  Add backend info to TypeInstance details. Minimal scope, with primitive fields which are available by default.

Screen Shot 2022-02-16 at 18 01 51

## Testing

```bash
docker run --rm -it -p 3000:8080 ghcr.io/capactio/pr/dashboard:PR-26
```

## Related issue(s)

- https://github.com/capactio/capact/issues/624

### CODE REVIEW

File: dashboard/src/typeinstances/TypeInstanceDetails.tsx [hunk 1]
```diff
@@ -65,6 +65,20 @@ function TypeInstanceDetails({
...
+              <strong>ID</strong>
+            </Col>
+            <Col span={20}>
+              <Text code>{typeInstance.backend.id}</Text>
+            </Col>
+            <Col span={4}>
+              <strong>Abstract</strong>
+            </Col>
+            <Col span={20}>{typeInstance.backend.abstract.toString()}</Col>
+          </Row>
```
[the remark this answers is not in the document]
Comment (mszostok) [reply dashboard/src/typeinstances/TypeInstanceDetails.tsx:80]: opt1 is ok 👍

**Comment by pkosiec** [comment]
1. I think it should be that because we had the precommit hook setup broken for a while (as you also noticed). Unfortunately I forgot to run the `prettify` command for all files in my (already quite invasive) PR. Anyway, the script which set ups precommit hook is fixed now, so it should be one-time change. You can keep the formatted files on your PR, or we can run `prettify` directly on `main`. Up to you, I don't have problem with both options 🙂 
2. It could be because of different Node & npm version (I have `npm: '8.3.1', node: '17.4.0'`). Don't worry about that, you can commit this lock file. I hope Lerna with hoisting enabled is predictable... should be 🤞

File: .prettierignore [hunk 2]
```diff
@@ -2,3 +2,5 @@ generated
 coverage
 build
 dist
+package-lock.json
+node_modules
```
Comment (pkosiec) [inline .prettierignore:6]: Please also add a directory with Storybook build output. Tested that locally and it works. Thanks!

```suggestion
node_modules
storybook-static
```

**Review by pkosiec — APPROVED** [review]
👌

**Pull request merged**